src/components/Search.js

```diff
@@ -28,9 +30,10 @@ const Search = props => (
                 />
               </td>
               <td>
-                <button className="btn btn-primary fn-submit-name">Search IDs</button>
-              </td>
+                {<Button name = "Search IDs"/>}
+              </td> 
               {/* Add search by name here */}
+              <SearchByName />
```

Comment:
Nice job! Do you think it would be better to also extract a `SearchByCustomerId` component?